gravitee/templates/gateway-ingress.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     - host: {{ $host }}
       http:
         paths:
-          - path: /
+          - path: /gateway
```

Comment:
not sure why you force a path for the gateway.

Reply:
It's because we're using Nginx Ingress Controller with a single domain and multiple routes (for the gateway, mgmt API and the portal). It's to allow serving the APIM through the Ingress Controller using HTTPS as well (that way we keep a single domain and use a single certificate).